Jenkinsfile

```diff
@@ -802,8 +802,7 @@ pipeline {
                         }
                         */
                         always {
-				sh '''python utils/fix_cmocka_xml.py
-					ls -l test_results/'''
+				sh '''python utils/fix_cmocka_xml.py'''
```

Comment:
Technically only need one single quote here rather than the 3 for multi-line strings.

Reply:
Fixed the strings.